### PR TITLE
[PFS] Start pfs in foreground and log in terminal and file in debug mode

### PR DESCRIPTION
**Pull request by YingChen1996**
# Description
- pf service start --debug will start pfs in forground and print log in both terminal and pfs.log
image

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by wangchao1230 — APPROVED** [review]

File: src/promptflow-devkit/promptflow/_cli/_pf/_service.py [hunk 1]
```diff
@@ -157,42 +157,35 @@ def start_service(args):
...
-                app, _ = create_app()
-            if os.environ.get(PF_SERVICE_DEBUG) == "true":
-                app.logger.setLevel(logging.DEBUG)
-            else:
-                app.logger.setLevel(logging.INFO)
-            message = f"Starting Prompt Flow Service on {port}, version: {get_pfs_version()}."
-            app.logger.info(message)
-            print(message)
-            sys.stdout.flush()
+        if not is_run_from_built_binary():
```
Comment (wangchao1230) [inline src/promptflow-devkit/promptflow/_cli/_pf/_service.py:160]: add more description to the help message on --debug param

File: src/promptflow-devkit/promptflow/_cli/_pf/_service.py [hunk 2]
```diff
@@ -98,15 +98,23 @@ def add_parser_start_service(subparsers):
         action="store_true",
         help="If the port is used, the existing service will be terminated and restart a new service.",
     )
+    add_param_debug = lambda parser: parser.add_argument(  # noqa: E731
+        "-d",
+        "--debug",
+        action="store_true",
+        help="The debug mode flag for the promptflow service activates foreground operation, displaying debug level "
```
Comment (wangchao1230) [inline src/promptflow-devkit/promptflow/_cli/_pf/_service.py:105]: make it simpler: 

Start the promptflow service in foreground, displaying debug level logs directly in the terminal

**Review by wangchao1230 — APPROVED** [review]

**Review by wangchao1230 — APPROVED** [review]

File: src/promptflow-devkit/promptflow/_sdk/_service/app.py [hunk 3]
```diff
@@ -99,8 +101,19 @@ def create_app():
         handler = RotatingFileHandler(filename=log_file, maxBytes=1_000_000, backupCount=1)
         formatter = logging.Formatter("[%(asctime)s][%(name)s][%(levelname)s] - %(message)s")
         handler.setFormatter(formatter)
+
```
Comment (Stephen1993) [inline src/promptflow-devkit/promptflow/_sdk/_service/app.py:104]: Are these handler codes used in multiple places? If so, please package it as a function.

Comment (YingChen1996) [reply src/promptflow-devkit/promptflow/_sdk/_service/app.py:104]: It's used in pfs for now

**Review by Stephen1993 — APPROVED** [review]

**Pull request merged**